### PR TITLE
Update to Sierra version of Jitouch

### PR DESCRIPTION
**Pull request by tangestani**

### CODE REVIEW

File: Casks/jitouch.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   version :latest
   sha256 :no_check
 
-  url 'https://www.jitouch.com/jitouch_el_capitan.zip'
```
Comment (jawshooah) [inline Casks/jitouch.rb]:
```rb
if MacOS.version <= :el_capitan
  url 'https://www.jitouch.com/jitouch_el_capitan.zip'
else
  url 'https://www.jitouch.com/jitouch_sierra.zip'
end
```

**Review by jawshooah — CHANGES_REQUESTED** [review]

File: Casks/jitouch.rb (hunk 1 above)
Comment (tangestani) [inline Casks/jitouch.rb]: Good idea. Thanks for the tip. 👍

**Comment by jawshooah** [comment]
You didn't follow the steps listed in the PR template. Namely, you didn't run `brew cask style Casks/jitouch.rb`: https://travis-ci.org/caskroom/homebrew-cask/jobs/168863351#L426

You can run `brew cask style --fix Casks/jitouch.rb` to fix those issues.

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 3a61af920a2582ba552d10dbae118905ff314362. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**